content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
+
+## Security
+
+- **Added** Enabled the workload cert rotate automatically in VM
+ ([Issue #24554](https://github.com/istio/istio/issues/24554))
+
+- **Added** support for creation of CSRs using ECC based certificates.
+ ([Issue #23226](https://github.com/istio/istio/issues/23226))
+
+- **Added** SDS support for Client Certificate and CA certificate used for TLS/mTLS Origination from Egress Gateway.
+  ([Issue #14039](https://github.com/istio/istio/issues/14039))
+
+- **Improved** Trust Domain Validation to validate TCP traffic as well, previously only HTTP traffic was validated
+
+- **Improved** Istio Gateways to allow use of source principal based authorization when the Server's TLS mode is `ISTIO_MUTUAL`.
```

Comment:
Can we add the tracking bug or PR to the end of this?
@yangminzhu

Reply:
https://github.com/istio/istio/pull/25818

Reply:
Should we be marking it as improved? when we have a CVE release note for this?

Reply:
Added the PR link (although ideally we should have an issue link).

Leaving this conversation open because I'm not sure what to do about @rshriram comment above.

Reply:
I would mark it as Fixed because there is a CVE associated with it with score of 5.9.. And how do we link to the CVE here ?

Reply:
I think we should call out that it was CVE with score that Shriram mentioned.

Reply:
I think it should be "Fixed". @yangminzhu for the CVE release note link.

Reply:
I think the problem here is we don't have any announcement (at least not on istio.io) for the CVE: https://istio.io/latest/news/security/